sentry-android-core/src/main/java/io/sentry/android/core/internal/gestures/SentryGestureListener.java

```diff
@@ -243,6 +244,8 @@ private void startTracing(final @NotNull UiElement target, final @NotNull String
         hub.startTransaction(
             new TransactionContext(name, TransactionNameSource.COMPONENT, op), transactionOptions);
 
+    transaction.getSpanContext().setOrigin(TRACE_ORIGIN + "." + eventType);
```

Comment:
`h`: I think what I'd like to see instead of `auto.ui.gesture_listener.click` is rather `auto.ui.gesture_listener.old_view_system` and `auto.ui.gesture_listener.jetpack_compose`, to understand where this span is coming from. I think to achieve that you could add a new property `origin` to [UiElement](https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/internal/gestures/UiElement.java) and then set it respectively for Views/JPC and use here.